test/functional/fundrawtransaction.py

```diff
@@ -449,12 +449,12 @@ def run_test(self):
         ############################################################
         # locked wallet test
         self.nodes[1].encryptwallet("test")
+        self.stop_node(0)
```

Comment:
Maybe move these up one line so nodes[1].encrypt and pop(1) can be next to each other.